src/transform/CollapseTable.js

```diff
@@ -5,6 +5,15 @@ import Polyfill from './Polyfill'
 
 const SECTION_TOGGLED_EVENT_TYPE = 'section-toggled'
 const BREAKING_SPACE = ' '
+const CLASS = {
```

Comment:
Thank you for symbolizing these! These names are dangerously inconsistent, e.g., `_collapse` and `_collapsed`, but far less dangerous when hidden behind a constant.